wp-otis.php

```diff
@@ -100,46 +100,64 @@ function wp_otis_acf_json_load_point( $paths ) {
 } );
 
 if ( ! wp_next_scheduled( 'wp_otis_expire_events' ) ) {
-	wp_schedule_event( time(), 'daily', 'wp_otis_expire_events' );
+  wp_schedule_event( time(), 'daily', 'wp_otis_expire_events' );
 }
 
 add_action( 'wp_otis_expire_events', function () {
-	$logger = new Otis_Logger_Simple();
-
-	$query = new WP_Query( array(
-		'posts_per_page' => -1,
-		'post_type'      => 'poi',
-		'orderby'        => 'meta_value',
-		'order'          => 'ASC',
-		'meta_query'     => array(
-			array(
-				'key'     => 'end_date',
-				'type'    => 'DATE',
-				'value'   => date( 'Y-m-d' ),
-				'compare' => '<',
-			),
-		),
-		'tax_query'      => array(
-			array(
-				'taxonomy' => 'type',
-				'field'    => 'slug',
-				'terms'    => TROR_SLUG_EVENTS,
-			),
-		),
-	) );
-
-	while ( $query->have_posts() ) {
-		$query->the_post();
-
-		wp_update_post( [
-			'ID'          => get_the_ID(),
-			'post_status' => 'draft',
-		] );
-
-		$logger->log( 'Updated expired event (set status draft) with UUID: ' . get_field( 'uuid' ), get_the_ID() );
-	}
-
-	wp_reset_postdata();
+  $logger = new Otis_Logger_Simple();
+
+  $logger->log( 'Checking for expired posts -----------------------');
+
+  $query = new WP_Query( array(
+    'posts_per_page' => -1,
+    'post_type'      => 'poi',
+    'orderby'        => 'meta_value',
+    'order'          => 'ASC',
+    'meta_query'     => array(
+      'relation' => 'AND',
+      array(
+        'post_status'    => 'published',
```

Comment:
`post_status` is a filter on the main WP_Query object, not an option on the `meta_query` filter.
```php
$query = new WP_Query( array(
    'posts_per_page' => -1,
    'post_type'      => 'poi',
    'orderby'        => 'meta_value',
    'order'          => 'ASC',
    'post_status'    => 'published',
    'meta_query'     => array(
...
```